includes/wikia/Extensions.php

```diff
@@ -705,7 +705,7 @@
 }
 
 // Enable CategorySelect extension for all not RTL wikis
-if (!in_array($wgLanguageCode, array('ar', 'fa', 'he', 'ps', 'yi')) && is_null( $wgEnableCategorySelectExt) ) {
+if (!in_array($wgLanguageCode, array('ar', 'fa', 'he', 'ps', 'yi')) && !isset($wgEnableCategorySelectExt) ) {
```

Comment:
I believe this won't work if `wgEnableCategorySelectExt` is explicitly set to `false` in WikiFactory.

Reply:
```php
> $a=false;

> var_dump( isset($a))
bool(true)
```

Reply:
Gotta love PHP. So `empty()` perhaps?

Reply:
My bad, I misremembered—this happens when you set an array entry to `null`, then it'll be considered "not set".